src/main/resources/google_checks.xml

```diff
@@ -11,6 +11,7 @@
     http://checkstyle.org (or in your downloaded distribution).
 
     To completely disable a check, just comment it out or delete it from the file.
+    To suppress certain violations please review suppression filters.
```

Comment:
As of right now, there is no way for users to know how to use suppression in custom checks without looking at the file and seeing the property name to be used. What do you think of expanding the google/sun reports to specify how to use the suppression. I would think a short description, specifying the property names and pointing to the other filter documentation would be enough.

Reply:
lets me finish with examples of usage .... and based on this we can think what words are better to use....

Reply:
here are examples in maven - https://github.com/sevntu-checkstyle/checkstyle-samples/pull/23
looks good
executed as (8.28-SNAPSHOT is based on this branch code):
```
✔ ~/java/github/sevntu-checkstyle/checkstyle-samples [google-samples-suppression L|✔] 
$ mvn -f pom-google.xml clean checkstyle:checkstyle -Dcheckstyle.version=8.28-SNAPSHOT

$ mvn -f pom-google-custom-suppression.xml clean checkstyle:checkstyle -Dcheckstyle.version=8.28-SNAPSHOT

```

Reply:
done.